api/provider/aws/certificates.go

```diff
@@ -147,6 +147,28 @@ func (p *AWSProvider) CertificateList() (structs.Certificates, error) {
 		})
 	}
 
+	// only fetch ACM certificates in regions that support it
+	switch os.Getenv("AWS_REGION") {
+	case "us-east-1":
+		c, err := p.certificateListACM()
+
```

Comment:
@ddollar style tip... it seems like it's more idiomatic to put the error check immediately on the next line.  This helps another developer quickly see that errors are handled